jsconfig.json

```diff
@@ -3,6 +3,7 @@
 		"allowJs": true,
```

Comment:
```suggestion
```
Nit [non-blocking]: `"allowJs": true` is implicit in the use of `jsconfig.json`

Reply:
Not to detract from an otherwise excellent-looking PR!

Reply:
Good spot! Thanks :)